package.json

```diff
@@ -14,7 +14,7 @@
     "console:ropsten": "npx hardhat console --network ropsten",
     "clean": "forge clean && rm -rf artifacts",
     "test": "forge test --no-match-contract IntegrationTest",
-    "test:integration": "FORK_BLOCK=14516592; forge test --fork-url https://eth-mainnet.alchemyapi.io/v2/$MAINNET_ALCHEMY_API_KEY --fork-block-number $FORK_BLOCK --match-contract IntegrationTest",
+    "test:integration": "FORK_BLOCK=14726759; forge test --fork-url https://eth-mainnet.alchemyapi.io/v2/$MAINNET_ALCHEMY_API_KEY --fork-block-number $FORK_BLOCK --match-contract IntegrationTest",
```

Comment:
It's weird to me that we're setting the fork block in two places - here and in the hardhat config. Should we use a commited .env? (for example - .forksettings.env or something)

Reply:
That was the earlier purpose behind my committed `block.txt` file :)

Reply:
As discussed, added the `block.txt` and we'll source it. Bad practice imo to commit `.env` files + I use a global gitignore for them